src/models/ITestResult.ts

```diff
@@ -88,6 +88,13 @@ export interface TestType {
   modificationTypeUsed?: string | null;
   particulateTrapFitted?: string | null;
   fuelType?: string | null;
+  centralDocs?: CentralDocs;
+}
+
+export interface CentralDocs {
+  issueRequired: boolean;
+  notes?: string;
+  reasonsForIssue?: string[];
```

Comment:
Is there any reason that we make this optional, why not a mandatory property when `centralDocs` exists, but default to an empty array.

It makes the code more predictable if we are not working with null-ish arrays.

Reply:
The requirement on the ticket is that the keys are optional

Reply:
Will update it now, to be mandatory and default to an empty array